src/main/java/org/apache/fineract/utils/CsvUtility.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.fineract.utils;
+
+import org.apache.fineract.exception.WriteToCsvException;
+import javax.servlet.http.HttpServletResponse;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+public class CsvUtility {
+
+    public static <T> void writeToCsv(HttpServletResponse response, List<T> listOfData) throws WriteToCsvException {
+        response.setContentType("text/csv");
+        DateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss");
+        String currentDateTime = dateFormatter.format(new Date());
+        String filename = "transactionRequest_" + currentDateTime + ".csv";
```

Comment:
oops, I forgot the filename.

Reply:
File name can be generic as well or take it as an argument